files.js

```diff
@@ -41,21 +64,25 @@ function loadTypedArrayFile(filename, Type, cb) {
     .catch(cb)
 }
 
-function saveBitsetFile(filename, seq, bitset, cb) {
+function saveBitsetFile(filename, version, seq, bitset, cb) {
   bitset.trim()
-  saveTypedArrayFile(filename, seq, bitset.count, bitset.words, cb)
+  saveTypedArrayFile(filename, seq, version, bitset.count, bitset.words, cb)
```

Comment:
here is the problem :)

Reply:
aaaah

Reply:
weird that tests don't break, I'll try adding a failing test first